### PR TITLE
Adding info about the Java SDK's StaticMapCriteria class

### LINKED ISSUE #440: Add writing about StaticMapCriteria class
Related ticket https://github.com/mapbox/mapbox-java/issues/802

@osana told me about [the `StaticMapCriteria` class](https://github.com/mapbox/mapbox-java/blob/afe9e88c9a09a413405571678d17499aa0a5f25c/services-staticmap/src/main/java/com/mapbox/api/staticmap/v1/StaticMapCriteria.java) in `mapbox-java` . We should add docs about this class and the constant strings for the various map styles

### PR DESCRIPTION
**Pull request by langsmith**
Resolves #440  by adding a little more info about the `StaticMapCriteria` convenience class.

screencapture-localhost-8080-android-docs-java-overview-static-image-2018-11-29-17_23_45

### CODE REVIEW

File: src/pages/java/overview/static-image.md [hunk 1]
```diff
@@ -46,7 +46,11 @@ val staticImage = MapboxStaticMap.builder()
 />
 }}
 
-Make sure to use the `StaticMapCriteria` to reference the default Mapbox map styles such as Light, Dark, or Streets. The `MapboxStaticMap` image generation will not work correctly if you try to retrieve the styles' strings from the `Style` class that lives in the Mapbox Maps SDK for Android.
+## Criteria
+
+[The `StaticMapCriteria` class](https://github.com/mapbox/mapbox-java/blob/master/services-staticmap/src/main/java/com/mapbox/api/staticmap/v1/StaticMapCriteria.java) holds private constant final `String`s which you can conveninetly reference as you build the `MapboxStaticMap.builder()` object. Using the `StaticMapCriteria` class will ensure that you will pass the correct parameters. Be sure not to use the Maps SDK's `Style` class to retrieve a style URL. You must use `StaticMapCriteria`.
```
Comment (colleenmcginnis) [inline src/pages/java/overview/static-image.md]: conveninetly --> conveniently

[the remark this answers is not in the document]
Comment (langsmith) [reply src/pages/java/overview/static-image.md]: Fixed

Comment (langsmith) [reply src/pages/java/overview/static-image.md]: Fixed

**Comment by langsmith** [comment]
>If we reword it like this is it still true?

Yep, still true. We'll go with your version ✍️

**Comment by langsmith** [comment]
Ok @colleenmcginnis , review tweaks have been made

**Pull request merged**